packages/main/src/themes/Button.css

```diff
@@ -240,10 +240,15 @@ bdi {
 }
 
 :host([design="Emphasized"][focused]) .ui5-button-root:after {
-	border-color: var(--sapContent_ContrastFocusColor);
+	border-color: var(--_ui5_button_emphasized_focused_border_color);
 	outline: none;
 }
 
+/* Belize related */
```

Comment:
I think the focus border on Down State should have this parameter for all designs, and not only Emphasized one.

Now:
![image](https://github.com/SAP/ui5-webcomponents/assets/88034608/78efe73f-7714-4071-aa95-e1466655633b)

Expected:
![image](https://github.com/SAP/ui5-webcomponents/assets/88034608/6d133798-2e4f-4d92-b618-7c47800ab5e9)

Reply:
Is this something introduced with this change? Cause this is not part of the reported bug.

Reply:
No, it is actually not. I got mistaken by the name of the PR. Not valid.